### ISSUE TITLE
Analyze and fix warning around expected directories during rorpiter tests

### ISSUE THREAD

**Issue by ericzolf** [rdiff-backup/rdiff-backup#81]
<a href="https://github.com/ericzolf"><img src="https://avatars3.githubusercontent.com/u/3105685?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ericzolf](https://github.com/ericzolf)**
_Saturday Jun 01, 2019 at 15:25 GMT_
_Originally opened as https://github.com/ericzolf/rdiff-backup/issues/27_

----

The tests are successful but there are warnings (under all Python versions) about missing directories in the backup output directory:

```
py38 runtests: commands[20] | python testing/rorpitertest.py
...Warning: expected /home/ericl/Public/rdiff-backup/.tox/py38/testfiles/output/3 to be a directory but found None instead.
This is probably caused by a bug in versions 1.0.0 and earlier.
Warning: expected /home/ericl/Public/rdiff-backup/.tox/py38/testfiles/output/3/4 to be a directory but found None instead.
This is probably caused by a bug in versions 1.0.0 and earlier.
```

The reference to the bug in version 1.0.0 is strange because the backup directory is created from scratch during the tests (with what will become version 1.3.4), hence I think it can't be that easily ignored.

**Comment by janvlug** [comment]
rdiff-backup cannot make an incremental backup any more. See the attached file:
[2019-12-08-error.txt](https://github.com/rdiff-backup/rdiff-backup/files/3936637/2019-12-08-error.txt)
I'm backing up to an external disk, and sometimes the system is suspended during the rdiff-backup. Maybe this could be related.

**Comment by ericzolf** [comment]
Closed by #676
